### PR TITLE
Add more travis tests and update coarseDataTools remote

### PR DESCRIPTION
**Pull request by zkamvar**
I just got a positive confirmation from the developer of coarseDataTools that he will merge the package into master and submit it to cran, so I am removing the hackout3 branch here.

I'm also getting travis to run on different architectures and report coverage (when set up)

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ fitRotavirus.R
 ^.*\.Rproj$
 ^\.Rproj\.user$
 ^\.travis.yml
+^\.travis\.yml$
```
Comment (jstockwin) [inline .Rbuildignore]: This line should probably just replace the one above? (They're both about `.travis.yml`)

File: .travis.yml [hunk 2]
```diff
@@ -1,2 +1,35 @@
...
+
+warnings_are_errors: true
+
+notifications:
+  email:
+    on_success: change
+    on_failure: change
+
+after_success:
+  - if [[ "${R_CODECOV}" ]]; then Rscript -e 'covr::codecov()'; fi
```
Comment (jstockwin) [inline .travis.yml]: What do we need to do to set this up? Might as well get the reporting in there straight away?

File: DESCRIPTION [hunk 3]
```diff
@@ -1,6 +1,5 @@
 Package: EpiEstim
-Version: 2.1-0
-Date: 2012/01/17
+Version: 2.0-0
```
Comment (jstockwin) [inline DESCRIPTION]: Version is going backwards..?

Comment (zkamvar) [reply DESCRIPTION]: sorry, that was a mistake of merging the files from different branches.

File: .Rbuildignore (hunk 1 above)
Comment (zkamvar) [reply .Rbuildignore]: Indeed it should!

File: .travis.yml (hunk 2 above)
Comment (zkamvar) [reply .travis.yml]: Actually, you're right. We don't need to set up codecov. It will just work:tm: when pushed from travis.

Comment (jstockwin) [reply .travis.yml]: Oh cool, didn't realise it just works :crossed_fingers:

**Pull request merged**